project/src/hooks/use-active-card-state.ts

```diff
@@ -0,0 +1,13 @@
+import {useState} from 'react';
+
+type ActivityCardType = (id: string) => void;
+
+const useActiveFilmCard = (): [ActivityCardType] => {
+  const [, setFilmCardActive] = useState<string>();
+  function toggleActiveCardById(id: string) {
```

Comment:
в других местах, о тебя стрелочные функции
```suggestion
  const handleActiveCardByIdToggle = (id: string) => {
```